lib/legacy/application.ex

```diff
@@ -7,7 +7,11 @@ defmodule Goth.Application do
     envs = Application.get_all_env(:goth)
 
     if envs == [] do
-      Supervisor.start_link([], strategy: :one_for_one)
+      children = [
+        {Registry, keys: :unique, name: GothRegistry}
```

Comment:
```suggestion
        {Registry, keys: :unique, name: Goth.Registry}
```

Reply:
Same thing, let's go for `Goth.Registry`